velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -404,5 +408,36 @@ TEST_F(DateTimeFunctionsTest, dateDiffDate) {
       dateDiff(parseDate("-5877641-06-23"), parseDate("1994-09-12")));
 }
 
+TEST_F(DateTimeFunctionsTest, addMonths) {
+  const auto parseDate = [](const std::string& dateString) {
+    return DATE()->toDays(dateString);
+  };
+
+  const auto addMonths = [&](const std::string& dateString, int32_t value) {
+    return evaluateOnce<int32_t, int32_t>(
+        "add_months(c0, c1)",
+        {parseDate(dateString), value},
+        {DATE(), INTEGER()});
+  };
+
+  EXPECT_EQ(addMonths("2015-01-30", 1), parseDate("2015-02-28"));
+  EXPECT_EQ(addMonths("2015-01-30", 11), parseDate("2015-12-30"));
+  EXPECT_EQ(addMonths("2015-01-01", 10), parseDate("2015-11-01"));
+  EXPECT_EQ(addMonths("2015-01-31", 24), parseDate("2017-01-31"));
+  EXPECT_EQ(addMonths("2015-01-31", 8), parseDate("2015-09-30"));
+  EXPECT_EQ(addMonths("2015-01-30", 0), parseDate("2015-01-30"));
+  EXPECT_EQ(addMonths("2016-03-30", -1), parseDate("2016-02-29"));
+  EXPECT_EQ(addMonths("2015-01-30", -2), parseDate("2014-11-30"));
+  EXPECT_EQ(addMonths("2015-04-20", -24), parseDate("2013-04-20"));
+
+  constexpr int32_t kMin = std::numeric_limits<int32_t>::min();
```

Comment:
Define these outside of this test and re-use. These are copy-pasted in multiple places in this fiile.

Reply:
Just fixed. And removed all other redundant definition in this file.